api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -323,6 +323,12 @@ message Watchdog {
   // kill behavior. If not specified the default is 0 (disabled).
   google.protobuf.Duration kill_timeout = 3;
 
+  // Defines the maximum skew used to adjust the *kill_timeout* if *kill_timeout* is
```

Comment:
nit (we generally call this jitter everywhere else and the field references jitter)

```suggestion
  // Defines the maximum jitter used to adjust the *kill_timeout* if *kill_timeout* is
```

Also needs a release note.